src/searchRecap/story.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { withKnobs, boolean, text } from '@storybook/addon-knobs'
+import SearchRecap from './index'
+
+const stories = storiesOf('Widgets|SearchRecap', module)
+stories.addDecorator(withKnobs)
+
+stories.add('default', () => (
+  <SearchRecap
```

Comment:
Wrap in a <Section /> ?

Reply:
Done, also included in the SearchResults page (Tabs story)